src/main/java/org/opentripplanner/updater/bike_park/HslBikeParkDataSource.java

```diff
@@ -0,0 +1,137 @@
+package org.opentripplanner.updater.bike_park;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.vividsolutions.jts.geom.*;
+import org.opentripplanner.routing.bike_park.BikePark;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Load bike parks from the HSL Park and Ride API.
+ *
+ * @author hannesj
+ */
+public class HslBikeParkDataSource extends GenericJsonBikeParkDataSource{
```

Comment:
That is true, Same goes for `GenericJsonBikeParkDataSource` and some other classes, I can try to refactor it a bit so that we avoid the almost-duplication.